lib/openhab/core/configuration.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module OpenHAB
+  module Core
+    java_import org.openhab.core.config.core.Configuration
+
+    # Adds methods to core openHAB {org.openhab.core.config.core.Configuration} to make it more natural in Ruby
+    class Configuration
+      # inspect result is just the string representation
+      alias_method :inspect, :to_s
```

Comment:
I think I'd be more in favor of doing a proper method that outputs a ruby-style inspect. like just delegate to `to_hash`, like `OpenHAB::Core::Metadata::NamespaceHash` does

Reply:
So, leave #to_s as is (i.e. it will output `Configuration[{key=hostname; type=String; value=xxx}, {key=refreshInterval; type=BigDecimal; value=15}]`) and #inspect to output like a normal hash? I want to still be able to see that it's an actual java `Configuration` class somehow, wdyt?

Reply:
I've just delegated it to #to_h